src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partner = '';
+
+  if (person.sex === 'm') {
+    partner = 'wife';
+  } else {
+    partner = 'husbant';
```

Comment:
There is a typo in the word 'husband'. It is currently spelled as 'husbant'. Correct it to 'husband' to ensure proper display.